src/qaqc/01-organize_raw_data.Rmd

```diff
@@ -380,13 +386,13 @@ all_data_summary_stats_list <- map(all_data_summary_list, generate_summary_stati
 Add flagging functions for each df in all_data_summary_list
 
 ```{r}
-# flag addition function
+# flag addition function 
 # This function will be called inside of add_x_flags() functions (ex. add_field_flags())
 # This function will be used to simply add flags into the flag column
 add_flag <- function(df, condition_arg, description_arg) {
   df <- df %>% mutate(flag = case_when(
-    {{condition_arg}} ~ if_else(is.na(flag), description_arg, paste(flag, description_arg, sep = ";\n")),
-    TRUE ~ flag))
+    {{condition_arg}} ~ if_else(is.na(flag), paste(description_arg), paste(flag, description_arg, sep = ";\n")),
+    TRUE ~ flag)) # <- KW question: what is this part of your code doing? 
```

Comment:
case_when() evaluates for the first `TRUE` condition in your case when pipe. This just hard codes `TRUE` at the end so that if none of the previous conditions were met you just return what is already in the flag column.